en/services/high_latency.md

```diff
@@ -10,18 +10,18 @@ The protocol provides a heartbeat-like message ([HIGH_LATENCY2](#HIGH_LATENCY2))
 High latency links are expensive.
 In order to reduce traffic to the bare minimum, some of the fundamental assumptions of MAVLink are explicitly broken:
 - [HEARTBEAT](../messages/common.md#HEARTBEAT) messages are not emitted on the channel (either by the autopilot or GCS).
-
-  > **Note** The heartbeat is used to build MAVLInk routing tables between channels.
+  > **Note** The heartbeat is used to build MAVLink routing tables between channels.
     Commands addressed specifically to the high latency component may not be routed from another channel (i.e. you can connect to the component from a GCS directly, but not via a MAVLink router).
-- Broadcast messages from the network should not be automatically sent over the high latency channel.
+- Only the [COMMAND](./command) service messages should be sent over the high latency channel.
+- Ground stations should not send or receive [MISSION](./mission) items or [PARAMETER](./parameter) items over the high latency channel in order to prevent congestion.
 
 The other rules are essentially the same but there are some implications of the above changes:
-- Broadcast messages from the high latency channel should be routed to other nodes on the network as usual.
+- Messages from the high latency channel should be routed to other nodes on the network as usual.
   Note that this in reality most systems on a high latency network **only** send `HIGH_LATENCY2`.
-- Addressed messages should be sent over the high latency channel (in both directions) in accordance with the normal routing rules.
-  In practice the lack of `HEARTBEAT` means that addressed messages are unlikely to arrive, and hence be sent.
+- Most addressed messages will not be sent over the high latency link.
+- Parameters, waypoints, geofences and rally points will not be downloaded over a high latency channel.
```

Comment:
I might look at this wording when I get back from holiday in a couple of weeks. What I'd like to do is reword this as something like "should not by default be"